components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.carbon.identity.framework</groupId>
+        <artifactId>cors-mgt</artifactId>
+        <version>5.17.91-SNAPSHOT</version>
```

Comment:
Make sure to update all the new pom files to the latest version before merging the PR.